lib/ansible/cli/pull.py

```diff
@@ -93,6 +94,10 @@ def init_parser(self):
 
         self.parser.add_argument('args', help='Playbook(s)', metavar='playbook.yml', nargs='*')
 
+        # remove unused default options
+        self.parser.add_argument('--list-hosts', help=argparse.SUPPRESS, action=opt_help.UnrecognizedArgument)
+        self.parser.add_argument('--flush-cache', help=argparse.SUPPRESS, action=opt_help.UnrecognizedArgument)
```

Comment:
Alternatively, I could update ansible-pull to pass this through to ansible-playbook?

Reply:
the list does not make sense, but the flush does